app/main/controller/image_ocr_controller.py

```diff
@@ -37,9 +45,27 @@ def post(self):
         if not texts:
             return
```

Comment:
do we also want to log some thing here so we know how many images don't have text?  Or maybe we can already get that count from the post messages in logs?

Reply:
Good point, but I think we're fine not to log as we can calculate that from other data as you suggest